### PR TITLE
fix reboot.bin and some clean optimizations

### PR DESCRIPTION
**Pull request by Matsumot0**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -72,7 +73,7 @@ all-target-brahma:
 	$(MAKE) -C brahma
 
 reboot/reboot.bin:
-	$(MAKE) -C $(dir $@) $(notdir $@)
```
Comment (173210) [inline Makefile]: Can you explain this change>

**Comment by Matsumot0** [comment]
i don't remove this entire line, but just $(notdir $@) variable to avoid  compilation error on reboot.bin file due of nastys change nothing else.

**Comment by 173210** [comment]
I looked at reboot/Makefile. The crappy devkitPro format seems to prevent the recursive make. Anyway, the change makes sense now.

**Comment by Matsumot0** [comment]
:+1:

**Pull request merged**